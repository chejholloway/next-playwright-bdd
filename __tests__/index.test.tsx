import { render } from '@testing-library/react';
import HomePage from '../src/pages/index';

describe('<HomePage>', () => {
    it('should render component', () => {
        const { asFragment } = render(<HomePage />);
        expect(asFragment()).toMatchSnapshot();
    });
});
