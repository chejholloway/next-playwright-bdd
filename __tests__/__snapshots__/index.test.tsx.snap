// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage> should render component 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="text-3xl font-bold underline"
    >
      Hello world!
    </h1>
    <div
      class="count"
    >
      <h1
        data-testid="counter-text"
      >
        Count: 0
      </h1>
    </div>
    <div
      class="app"
    >
      <div
        class="buttons"
      >
        <button
          data-testid="decrease"
          title="-"
          type="button"
        >
          -
        </button>
        <button
          data-testid="increase"
          title="+"
          type="button"
        >
          +
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
